Array/src/ArrayTask4.java

```diff
@@ -0,0 +1,86 @@
+            enterWords = sc.nextLine();
+            words = enterWords.split(",");
+            if (!enterWords.contains(",") || words.length < 3) {
+                System.out.println("Something wrong! You should enter 3 or more words separated by commas without gaps ");
+            } else {
+                end = true;
+            }
+        } while (!end);
+        do {
+            System.out.println("Choose an action with an array of words and enter a number: ");
+            System.out.println("1 - maximum length ");
+            System.out.println("2 - start with");
+            System.out.println("3 - end with");
+            System.out.println("4 - contains");
+            System.out.println("0 - exit");
+            userAction = sc.next();
+            findValue = 0;
+            switch (userAction) {
+                case "1" -> {
+                    System.out.println("Enter the integral for the length word");
+                    enterInt = sc.nextInt();
+                    System.out.println("Result: ");
+                    for (String i : words) {
+                        if (i.length() <= enterInt) {
+                            System.out.println(++findValue + " " + i);
+                        }
+                    }
+                    if (findValue == 0) System.out.println("Did not find any value");
+                }
+                case "2" -> {
+                    System.out.println("Enter the letter for the start with word");
+                    enterValue = sc.next();
+                    System.out.println("Result: ");
+                    for (String i : words) {
+                        if (i.startsWith(enterValue)) {
+                            System.out.println(++findValue + " " + i);
+                        }
+                    }
+                    if (findValue == 0) System.out.println("Did not find any value");
+                }
+                case "3" -> {
+                    System.out.println("Enter the letter for the end with word");
+                    enterValue = sc.next();
+                    System.out.println("Result: ");
+                    for (String i : words) {
+                        if (i.endsWith(enterValue)) {
+                            System.out.println(++findValue + " " + i);
+                        }
+                    }
+                    if (findValue == 0) System.out.println("Did not find any value");
+                }
+                case "4" -> {
+                    System.out.println("Enter the letters for the part of word");
+                    enterValue = sc.next();
+                    System.out.println("Result: ");
+                    for (String i : words) {
+                        if (i.contains(enterValue)) {
+                            System.out.println(++findValue + " " + i);
+                        }
+                        if (findValue == 0) System.out.println("Did not find any value");
```

Comment:
Please rewrite this function. Please move "if (findValue == 0)" outside of loop.

Reply:
Done

Reply:
Fine, thanks